docs/tutorials/azure-quantum/getting_started_ionq.ipynb

```diff
@@ -0,0 +1,449 @@
+   "outputs": [],
+   "source": [
+    "from azure.quantum.cirq import AzureQuantumService\n",
+    "service = AzureQuantumService(\n",
+    "    resource_id=\"\",\n",
+    "    location=\"\",\n",
+    "    default_target=\"ionq.simulator\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7e61d96a",
+   "metadata": {},
+   "source": [
+    "### List all targets\n",
+    "\n",
+    "You can now list all the targets that you have access to, including the current queue time and availability."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "b1ffab14",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<Target name=\"ionq.qpu\", avg. queue time=19 s, Available>,\n",
+       " <Target name=\"ionq.simulator\", avg. queue time=0 s, Available>,\n",
+       " <Target name=\"honeywell.hqs-lt-s1\", avg. queue time=0 s, Unavailable>,\n",
+       " <Target name=\"honeywell.hqs-lt-s1-apival\", avg. queue time=0 s, Available>,\n",
+       " <Target name=\"honeywell.hqs-lt-s2\", avg. queue time=93509 s, Available>,\n",
+       " <Target name=\"honeywell.hqs-lt-s2-apival\", avg. queue time=0 s, Available>,\n",
+       " <Target name=\"honeywell.hqs-lt-s1-sim\", avg. queue time=26 s, Available>]"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "service.targets()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "09ce82d3",
+   "metadata": {},
+   "source": [
+    "## Run a simple circuit\n",
+    "\n",
+    "Let's create a simple Cirq circuit to run. This circuit uses the square root of X gate, native to the IonQ hardware system."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "b97452a3",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<pre style=\"overflow: auto; white-space: pre;\">0: ───X^0.5───@───M(&#x27;b&#x27;)───\n",
+       "              │   │\n",
+       "1: ───────────X───M────────</pre>"
+      ],
+      "text/plain": [
+       "0: ───X^0.5───@───M('b')───\n",
+       "              │   │\n",
+       "1: ───────────X───M────────"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import cirq\n",
+    "\n",
+    "q0, q1 = cirq.LineQubit.range(2)\n",
+    "circuit = cirq.Circuit(\n",
+    "    cirq.X(q0)**0.5,             # Square root of X\n",
+    "    cirq.CX(q0, q1),              # CNOT\n",
+    "    cirq.measure(q0, q1, key='b') # Measure both qubits\n",
+    ")\n",
+    "circuit"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e2bcbea7",
+   "metadata": {},
+   "source": [
+    "You can now run the program via the Azure Quantum service and get the result. The following cell will submit a job that runs the circuit with 100 shots, wait until the job is completed and return the results."
```

Comment:
nit: Shots is a common term for this, I would recommend using repetitions or samples in order to stay consistent with parameter names.